xml/gl.xml

```diff
@@ -5674,11 +5674,11 @@ typedef unsigned int GLhandleARB;
         <enum value="0x8FF9" name="GL_FLOAT16_VEC2_NV"/>
         <enum value="0x8FFA" name="GL_FLOAT16_VEC3_NV"/>
         <enum value="0x8FFB" name="GL_FLOAT16_VEC4_NV"/>
-        <enum value="0x8FFC" name="GL_DOUBLE_VEC2" group="GlslTypeToken,AttributeType,UniformType"/>
```

Comment:
Removal of `GlslTypeToken`, replaced with `AttributeType`. Could be breaking downstream, but suitable replacement. Should be fine.

Reply:
Also note the [commit](https://github.com/KhronosGroup/OpenGL-Registry/pull/520/commits/1404006e38ea285937592e8157fe21cfabec496b) description, specifically `GL_UNSIGNED_INT_ATOMIC_COUNTER`.
That's the only enum I'm unsure about in this commit.